mpu6050/mpu6050.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ "$EUID" -ne 0 ]
```

Comment:
Does this script really require root access. You can modify user's permissions and run i2c-dev utilities from user.
See https://lexruee.ch/setting-i2c-permissions-for-non-root-users.html